packer/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: vagrant-virtualbox vagrant-libvirt scf-cached
```

Comment:
Hmm, `scf-cached` isn't a valid target here. (It's being hit by my tab completion)

Reply:
Fixed.